core/src/test/java/com/exadel/aem/toolkit/core/injectors/models/TestModelChildOuter.java

```diff
@@ -0,0 +1,31 @@
+package com.exadel.aem.toolkit.core.injectors.models;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.ValueMapValue;
+
+import com.exadel.aem.toolkit.api.annotations.injectors.Child;
+
+@Model(adaptables = SlingHttpServletRequest.class, defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
+public class TestModelChildOuter {
```

Comment:
(cosmetic) Can we give it more specific name for example for outer: `TestModelChildRootResource` and for inner `TestModelChildSubResource`?